lib/robots/dor_repo/gis_assembly/normalize_data.rb

```diff
@@ -5,49 +5,98 @@ module DorRepo
     module GisAssembly
       class NormalizeData < Base
         def initialize
-          super('gisAssemblyWF', 'normalize-data', check_queued_status: true) # init LyberCore::Robot
+          super('gisAssemblyWF', 'normalize-data')
+        end
+
+        def perform_work
+          logger.debug "normalize-data working on #{bare_druid}"
+
+          rootdir = GisRobotSuite.locate_druid_path bare_druid, type: :stage
+          datafn = "#{rootdir}/content/data_EPSG_4326.zip"
+          if File.size?(datafn)
+            logger.info "normalize-data: #{bare_druid} found existing normalized data: #{File.basename(datafn)}"
+            return
+          end
+
+          File.umask(002)
+          flags = {
+            overwrite_prj: true,
+            tmpdir: Settings.geohydra.tmpdir
+          }
+
+          fn = "#{rootdir}/content/data.zip" # original content
+          logger.debug "Processing #{bare_druid} #{fn}"
+
+          # Determine file format
+          modsfn = "#{rootdir}/metadata/descMetadata.xml"
+          raise "normalize-data: #{bare_druid} is missing MODS metadata" unless File.size?(modsfn)
+
+          format = GisRobotSuite.determine_file_format_from_mods modsfn
+          raise "normalize-data: #{bare_druid} cannot determine file format from MODS" if format.nil?
+
+          # reproject based on file format information
+          if GisRobotSuite.vector?(format)
+            reproject_shapefile fn, modsfn, flags
+          elsif GisRobotSuite.raster?(format)
+            proj = GisRobotSuite.determine_projection_from_mods modsfn
```

Comment:
`proj` would be better as `projection_in_mods` or even `projection` -- I read it as "project"

Reply:
Fixed.